sdk/storage/storage-blob/src/generated/src/operations/appendBlob.ts

```diff
@@ -152,7 +152,6 @@ const createOperationSpec: coreHttp.OperationSpec = {
     Parameters.encryptionKey,
     Parameters.encryptionKeySha256,
     Parameters.encryptionAlgorithm,
-    Parameters.encryptionScope,
```

Comment:
Is this by design?

Reply:
Yes. 
As you can see, there were two of them in one spec. I think it's brought in by some auto-merging. They didn't affect the upper layer interfaces so failed to spot them.